Ladybird/AppKit/Utilities/Conversions.mm

```diff
@@ -86,4 +86,14 @@ NSPoint gfx_point_to_ns_point(Gfx::IntPoint point)
         static_cast<CGFloat>(point.y()));
 }
 
+Gfx::Color ns_color_to_gfx_color(NSColor* color)
+{
+    return { static_cast<u8>([color redComponent] * 255), static_cast<u8>([color greenComponent] * 255), static_cast<u8>([color blueComponent] * 255), static_cast<u8>([color alphaComponent] * 255) };
```

Comment:
Is this safe? Getting the rgb components of an NSColor that is not in the rgb space will crash. Do we know we will always get an rgb NSColor?

Also please follow the style of the rest of the file and put the casts one-per-line to help with reading this.

Reply:
You are right this is wrong, I now convert the NSColor to RGB with `colorUsingColorSpace:`. I now the Gfx::Color has conversion constructors but this was easier and now we support all the formats Apple supports :)